lib/main.coffee

```diff
@@ -52,6 +52,15 @@ module.exports =
         parameters.push '--format=' +
                         'R-{line}:{column}-{end_line}:{end_column}' +
                         '-{severity}-{origin}:{message}'
+        # check for existence of .coafile
+        if helpers.exec(@executable,
+                        parameters,
+                        {cwd: path.dirname(filePath)}) then (result) ->
+          if result == 2
+            atom.notifications.addError \
+              'Project does not contain .coafile!',
+              'detail': 'Please provide a .coafile.'
```

Comment:
Hi @EverWinter23 !
What was intended in #60 was `coala-atom` should run only if a `.coafile` is provided. Else, it should do nothing. 
In your implementation, a project without .coafile would be told `Please provide a .coafile` every time the user edits their file. In the current `master` branch, every time the user edits their file, they are shown `ERROR: Could not run Linter` . Have you checked if the bug was fixed by installing your forked repo in atom ?